README.md

```diff
@@ -40,6 +40,12 @@
 
 ## Installation
 
+### Pip
```

Comment:
pip is a kind of command line I think...

Reply:
refactored to `using pip`